meeting_minutes/221013_SARIF_TC_68.md

```diff
@@ -0,0 +1,154 @@
+| Johnathan  | Gilday    | Contrast Security | Voting Member           |
+| Mary       | Martin    | Microsoft         | Voting Member           |
+| Michael    | Fanning   | Microsoft         | Voting member           |
+| Nathan     | Baird     | Microsoft         | Voting Member           |
+| Paul       | Anderson  | Grammatech, Inc.  | Member                  |
+| Pavel      | Mikula    | Sonar             | Voting Member           |
+| Stefan     | Hagen     | Individual        | Secretary, taking notes |
+| Thanassis  | Avgerinos | ForAllSecure Inc  | Voting Member           |
+| Yekaterina | O'Neil    | Micro Focus       | Voting member           |
+
+## 1.3 Procedures for this meeting (Co-Chair David)
+
+## 1.4 Approval of agenda (Co-Chair David)
+
+* https://www.oasis-open.org/committees/download.php/70435/agenda_20221013.html
+
+Agenda was approved
+
+## 1.5 Approval of previous minutes (Co-Chair David)
+
+* [Minutes of 2022-09-29_Meeting #67](https://www.oasis-open.org/committees/document.php?document_id=70429&wg_abbrev=sarif)
+
+Minutes were approved
+
+## 1.6 Review of action items and resolutions (Secretary Stefan)
+
+* ACTION on David to keep pushing to close on the wikipedia page
+  * Michael continues the action
+* ACTION on Mary to continue to develop fixes scenario
+  * ONGOING
+* ACTION on Michael to provide additional references for the wikipedia
+  * COMPLETE
+* ACTION on Jeff to provide an initial seed page at github to foster the future roadmap / extension discussion
+  * ONGOING
+
+## 1.7 Identification of SARIF TC voting members (Co-Chair David)
+
+### 1.7.1 Prospective members attending their first meeting
+
+### 1.7.2 Members attaining voting rights at the end of this meeting
+
+* Charles Wilson
+* Paul Anderson
+
+### 1.7.3 Members losing voting rights if they have not joined this meeting by the time it ends
+
+### 1.7.4 Members who previously lost voting rights who are attending this meeting
+
+### 1.7.5 Members who have declared a leave of absence
+
+# 2. Future Meetings
+
+## 2.1 Future meeting schedule (Co-Chair Keaton)
+
+- Scheduled Teleconferences (Thursdays at 08:00 PT / 15:00 UTC for 1.5 hours)
+    ```
+    October 27
+    ```
+- Possible face-to-face meeting when pandemic permits
+
+# 3. Liaisons
+
+## 3.1 [OpenSSF](https://openssf.org/) Security Tools group
+
+> Liaison Group: ForAllSecure/Thanassis Avgerinos, Contrast Security/Jeff Williams, Microsoft/Michael Fanning
+
+* Report status and any communication from OpenSSF Security Tools
+  * Thanassis: no update as meeting was cancelled
+* Compose a liaison statement to OpenSSF Security Tools if appropriate
+  * No update
+
+# 4. Discussion
+
+## 4.1 Review status on finalizing SARIF 2.1 errata [List of next steps - github issue #509](https://github.com/oasis-tcs/sarif-spec/issues/509)
+
+* David: Addressed all comments from Yekatarina and all but one comment from Michael. Plans to move to Michael after final comment addressed.
```

Comment:
Yekatarina -> Yekaterina

Reply:
Sorry for that. Fixed in [ce67fe58](https://github.com/oasis-tcs/sarif-spec/pull/549/commits/ce67fe586a37d38c8397e89658476307bfd58e28)

Reply:
No worries :)